thirdeye/thirdeye-pinot/src/main/resources/com/linkedin/thirdeye/detector/onboard-notification-email-template.ftl

```diff
@@ -67,12 +67,28 @@
               </tr>
               <tr>
                 <td colspan="1" style="color: #606060; font-size: 14px; line-height: 20px;">
-                  Mailing list
+                  Mailing list (to)
                 </td>
                 <td colspan="1" style="color: #606060; font-size: 14px; line-height: 20px; font-weight: bold;">
                   ${recipients}
                 </td>
               </tr>
+              <tr>
+                <td colspan="1" style="color: #606060; font-size: 14px; line-height: 20px;">
+                  Mailing list (cc)
+                </td>
+                <td colspan="1" style="color: #606060; font-size: 14px; line-height: 20px; font-weight: bold;">
+                ${ccRecipients}
+                </td>
+              </tr>
+              <tr>
+                <td colspan="1" style="color: #606060; font-size: 14px; line-height: 20px;">
+                  Mailing list (bcc)

```

Comment:
BCC should not be in the email

Reply:
Thanks for the pointer. I have removed it from the email. 

By the way, until we bring in the concept of alert/subscription-group ownership, this can still be accessed from Thirdeye!